docs/api/migration-guides/local-simulators.mdx

```diff
@@ -0,0 +1,165 @@
+
+```python
+from qiskit.circuit.library import RealAmplitudes
+from qiskit.circuit import QuantumCircuit, QuantumRegister, ClassicalRegister
+from qiskit.quantum_info import SparsePauliOp
+from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+from qiskit_ibm_runtime.fake_provider import FakeManilaV2
+
+# Bell Circuit
+qc = QuantumCircuit(2)
+qc.h(0)
+qc.cx(0, 1)
+qc.measure_all()
+
+# Run the sampler job locally using FakeManilaV2
+fake_manila = FakeManilaV2()
+pm = generate_preset_pass_manager(backend=fake_manila, optimization_level=1)
+isa_qc = pm.run(qc)
+sampler = Sampler(backend=fake_manila)
+result = sampler.run([(isa_qc)]).result()
+```        
+
+### AerSimulator
+
+You can use local testing mode to use simulators from [Qiskit Aer](/verify/simulate-with-qiskit-aer), which allow you to model noise.
+
+Example without noise:
+
+```python
+from qiskit_aer import AerSimulator
+from qiskit.circuit.library import RealAmplitudes
+from qiskit.circuit import QuantumCircuit, QuantumRegister, ClassicalRegister
+from qiskit.quantum_info import SparsePauliOp
+from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+from qiskit_ibm_runtime import Session, SamplerV2 as Sampler
+
+# Bell Circuit
+qc = QuantumCircuit(2)
+qc.h(0)
+qc.cx(0, 1)
+qc.measure_all()
+
+# Run the sampler job locally using AerSimulator.
+# Session syntax is supported but ignored because simulators don't support sessions.
+aer_sim = AerSimulator()
+pm = generate_preset_pass_manager(backend=aer_sim, optimization_level=1)
+isa_qc = pm.run(qc)
+with Session(backend=aer_sim) as session:
+    sampler = Sampler(session=session)
+    result = sampler.run([(isa_qc)]).result()
+```
+
+To simulate with noise, specify a system (quantum hardware) and submit it to Aer.  Aer builds a noise model based on the calibration data from that system, and instantiates an Aer backend with that model.  If you prefer, you can [build a noise model](/verify/building_noise_models).
+
+Example with noise: 
+
+```python
+from qiskit_aer import AerSimulator
+from qiskit.circuit.library import RealAmplitudes
+from qiskit.circuit import QuantumCircuit, QuantumRegister, ClassicalRegister
+from qiskit.quantum_info import SparsePauliOp
+from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+
+# Bell Circuit
+qc = QuantumCircuit(2)
+qc.h(0)
+qc.cx(0, 1)
+qc.measure_all()
+
+service = QiskitRuntimeService()
+
+# Specify a system to use for the noise model
+real_backend = service.backend("ibm_brisbane")
+aer = AerSimulator.from_backend(real_backend)
+
+# Run the sampler job locally using AerSimulator.
+pm = generate_preset_pass_manager(backend=aer, optimization_level=1)
+isa_qc = pm.run(qc)
+sampler = Sampler(backend=aer)
+result = sampler.run([isa_qc]).result()
+```
+
+<span id="clifford-sim"></span>
+### Clifford simulation
```

Comment:
In this section you can tell people to see a more in depth example of clifford simulation with aer here: https://docs.quantum.ibm.com/verify/stabilizer-circuit-simulation